appinventor/appengine/src/com/google/appinventor/client/editor/youngandroid/palette/YoungAndroidPalettePanel.java

```diff
@@ -192,6 +200,25 @@ public void onChange(ChangeEvent event) {
       }
     });
 
+    /* User presses the slash key, the search text box is focused */
+    RootPanel.get().addDomHandler(new KeyDownHandler() {
+      @Override
+      public void onKeyDown(KeyDownEvent event) {
+        DesignToolbar designToolbar = Ode.getInstance().getDesignToolbar();
+        if (designToolbar.currentView == DesignToolbar.View.FORM) {
+          if (event.getNativeKeyCode() == 191 && !isSearchTextFocused) {
+            {
```

Comment:
I don't think this extra nested scope is needed.

Reply:
fixed